articles/hajimehoshi.re

```diff
@@ -0,0 +1,75 @@
+= @<tt>{gomobile}完全に理解した
+
+完全に理解したとは言っていない。
+
+== 著者について
+
+星一 (@<tt>{@hajimehoshi}) と申します。
+趣味でGoを使っています。
+代表作はEbiten@<fn>{ebiten}という2Dゲームライブラリです。
+Ebitenと@<tt>{gomobile}を使ったゲームをいくつか共同制作していて、2019年8月現在の代表作は「くまのレストラン@<fn>{bearsrestaurant}」です。
+
+//footnote[ebiten][@<href>{https://ebiten.org}]
+//footnote[bearsrestaurant][@<href>{https://daigostudio.com/bearsrestaurant}]
+
+== @<tt>{gomobile}
+
+@<tt>{gomobile}はAndroidおよびiOS向けのアプリケーションもしくはライブラリを出力するソフトウェアです。
+パッケージは@<tt>{golang.org/x/mobile}です。
+Go本体ではshared libraryもしくはarchiveしか吐くことができません。
+
+@<tt>{gomobile}コマンドは大雑把に2つのモードがあります。
+@<tt>{gomobile build}と@<tt>{gomobile bind}です。
+@<tt>{gomobile build}は単独のアプリケーションを出力します。
+
+TODO: gomobile buildとgomobile bindについて
+
+@<tt>{gomobile build}は簡単に完成されたアプリケーションを作れる一方、柔軟性に欠けます。
+@<tt>{gomobile build}で作られたアプリケーションはOpenGLサーフェス一個だけしかもっておらず、
+他のコンポーネントを入れることができません。またストアに必要な署名などのファイルを挿入するのは、
+不可能ではないにしろ困難です。
+@<tt>{gomobile build}だけで、ストアに載せる実用アプリケーションを作ることは現実的ではありません@<fn>{tenntenn}。
+実際我々が開発しているゲームでは@<tt>{gomobile bind}を活用しています。
+
+//footnote[tenntenn][実際に@<tt>{gomobile build}で作られたアプリケーションをストアに載せた例もあります: @<href>{https://www.slideshare.net/takuyaueda967/go-mobileandroid}]
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Line 34 sentence length(103) exceeds the maximum sentence length of 100.
Over 3 characters. (ja-technical-writing/sentence-length)